network-api/locale/fr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,227 @@
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-08-12 15:32-0700\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: .\network-api\networkapi\settings.py:364
```

Comment:
Hm, looks like the `--ignore=` params got… ignored, I assume because of the path separator (Win / vs Unix \). We will need to make that dynamic.

Reply:
Hm, is this a result of `makemessages`? because windows has supported `/` as path separate since literally day 1, so if we can tell Django to always use unix delimiters, we should be fine.

Reply:
Yeah, and it seems they fixed that very problem in Django directly in 2014? https://code.djangoproject.com/ticket/23298